inspirehep/modules/migrator/tasks.py

```diff
@@ -288,26 +297,28 @@ def migrate_chunk(chunk, broken_output=None, dry_run=False):
 
 @shared_task()
 def add_citation_counts():
-    from elasticsearch.helpers import bulk as es_bulk
-    from elasticsearch.helpers import scan as es_scan
-    from collections import Counter
+    index, doc_type = schema_to_index('records/hep.json')
 
     def get_records_to_update_generator(citation_lookup):
         for recid, citation_count in citation_lookup.iteritems():
-            yield {'_op_type': 'update',
-                   '_index': 'hep',
-                   '_type': 'record',
-                   '_id': recid,
-                   'doc': {'citation_count': citation_count}
-                   }
-
-    logger.info("Extracting all citations...")
+            try:
+                uuid = PersistentIdentifier.get('literature', recid).object_uuid
```

Comment:
I wonder if this has any performance implication... but otherwise sounds great :)